.github/workflows/build.yml

```diff
@@ -8,11 +8,38 @@ on:
     paths-ignore:
       - 'build.gradle'
       - 'bumpver.toml'
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+    inputs:
+      venue:
+        type: choice
+        description: Venue to deploy to
+        options:
+          - SIT
+          - UAT
+
 jobs:
-  build-api:
+  build:
     name: build, lint, and test API
     runs-on: ubuntu-latest
+    outputs:
+      deploy_env: ${{ steps.gradle-build.outputs.deploy_env }}
+      deploy_env_lower: ${{ steps.gradle-build.outputs.deploy_env_lower }}
```

Comment:
Are these used?

Reply:
The uppercase deploy env is used to select the correct environment on line 137, but it looks like the lowercase one is an artifact from before build environments.  I'll clean that up